board/health.h

```diff
@@ -55,4 +55,7 @@ typedef struct __attribute__((packed)) {
   uint8_t canfd_enabled;
   uint8_t brs_enabled;
   uint8_t canfd_non_iso;
+  uint32_t it0_call_rate;
```

Comment:
irq instead of it?

Reply:
ok makes sense, that's just how ST named it.